sections/multicolumn.liquid

```diff
@@ -16,7 +16,7 @@
   }
 {%- endstyle -%}
 
-<div class="multicolumn{% unless section.settings.background_style == 'none' and settings.text_boxes_border_thickness > 0 or settings.text_boxes_shadow_opacity > 0 %} background-{{ section.settings.background_style }}{% endunless %}{% if section.settings.title == blank %} no-heading{% endif %}">
+<div class="multicolumn{% unless section.settings.background_style == 'none' and settings.text_boxes_border_thickness > 0 or settings.text_boxes_shadow_opacity > 0 %} background-{{ section.settings.background_style }}{% endunless %}{% if section.settings.title == blank %} no-heading{% endif %} color-{{ section.settings.color_scheme }} gradient">
```

Comment:
Switching between `background-none` and `background-secondary` appears to only adjust section padding/margin and has no effect on the section's background color. Would the quick solution to remove one of those setting options?